main.rb

```diff
@@ -0,0 +1,22 @@
+MORSE_CODE = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+  '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+  '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+  '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+  '--..' => 'Z', '.----' => '1', '..---' => '2', '...--' => '3',
+  '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+  '---..' => '8', '----.' => '9', '-----' => '0'
+}.freeze
+
```

Comment:
- [ ] Kindly as it's requested on requirements, add the `decode` method with its proper functionality. Also, remember that it's necessary to add spaces between words 

![image](https://github.com/aima98/Decode-Morse-Message/assets/93630700/202430ed-13b4-4684-bf13-e7e2dc3b3893)